client/my-sites/themes/theme-options.js

```diff
@@ -49,9 +50,33 @@ const purchase = config.isEnabled( 'upgrades/checkout' )
 		} ),
 		getUrl: getThemePurchaseUrl,
 		hideForTheme: ( state, themeId, siteId ) => (
+			( config.isEnabled( 'jetpack/pijp' ) && isJetpackSite( state, siteId ) ) ||
+			! getCurrentUser( state ) ||
+			hasFeature( state, siteId, FEATURE_UNLIMITED_PREMIUM_THEMES ) ||
+			! isThemePremium( state, themeId ) ||
+			isThemeActive( state, themeId, siteId ) ||
+			isPremiumThemeAvailable( state, themeId, siteId )
+		)
+	}
+	: {};
+
+const upgradePlan = config.isEnabled( 'upgrades/checkout' )
+	? {
+		label: i18n.translate( 'Upgrade to activate', {
+			comment: 'label prompting user to upgrade the Jetpack plan to activate a certain theme'
+		} ),
+		extendedLabel: i18n.translate( 'Upgrade to activate', {
+			comment: 'label prompting user to upgrade the Jetpack plan to activate a certain theme'
+		} ),
+		header: i18n.translate( 'Upgrade on:', {
```

Comment:
:information_source: This string doesn't exist elsewhere without a **context** (or with a different one). You can probably drop the context. If needed, you can use a translator comment instead to clarify meaning.